medicalpanel/templates/medicalpanel/signup.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block page_title %}Dashboard{% endblock %}
+{% block content %}
+
+    <style>
+            .errorlist {
+                color: #FF0000;
+            }
+            .successMsg{
+                color: mediumspringgreen;
+            }
+    </style>
+    <div class="container">
+        <div  style="padding-top:110px;">
```

Comment:
inline css is also not a good option, move all the internal and inline css to seperate file